src/jupyter_contrib_nbextensions/nbextensions/load_tex_macros/main.js

```diff
@@ -0,0 +1,38 @@
+define(function(require, exports, module) {
+    var Jupyter = require('base/js/namespace');
+
+    function loadLatexUserDefs() { 
+        $.get('latexdefs.tex').done(function(data) {
+            data = data.replace(/^/gm, '\$\$\$').replace(/$/gm, '\$\$\$');
+            if ($('#latexdefs').length > 0) $('#latexdefs').remove();
+            $('body').append($('<div/>').attr('id', 'latexdefs').text(data));
+            console.log('latex_envs: loaded user LaTeX definitions latexdefs.tex');
+            onMarkdownCellRendering();
```

Comment:
I am not sure it works because function `onMarkdownCellRendering()` is not defined. It is defined in `latex_envs` and is global (which is not a good practice indeed), but in the absence of `latex_envs`  extension, this will not work.

Reply:
Hmmm, somehow it works for me even with `latex_envs` disabled.